backoffice/templates/backoffice/user.html

```diff
@@ -162,6 +155,16 @@ <h3>{% trans "Enrollments" %}</h3>
 
         <div class="col-md-12">
 
+            {% if userform.instance.is_active and not userform.instance.is_superuser and not userform.instance.is_staff %}
+                <form action="{% url 'masquerade-mask' %}" method="POST" target="_blank" style="display: inline">
+                    {% csrf_token %}
+                    <input type="hidden" name="mask_user" value="{{ userform.instance.username }}">
```

Comment:
I have no idea how usernames are handled for now, but I have seen that you can use user's email with masquerade, wouldn't it be a better option?

Reply:
I thought no because it is a personal data so the less I expose it the better.

Reply:
If usernames are built by a combination of first/last names, not really, but ok.

Reply:
in openedx usernames are free and a lot of users choose anonymous nicknames.